modules/develop/pages/data-transforms/how-transforms-work.adoc

```diff
@@ -14,7 +14,7 @@ Data transforms let you run common data streaming tasks, like filtering, scrubbi
 
 Data transforms use https://webassembly.org/[WebAssembly^] (Wasm) engines inside a Redpanda broker, allowing Redpanda to control the entire transform lifecycle. For example, Redpanda can stop and start transforms when partitions are moved or to free up system resources for other tasks.
 
-Data transforms take data from an input topic and map it to an output topic. For each topic partition, a leader is responsible for handling the data. Redpanda runs a virtual machine (VM) on the same CPU core (shard) as these partition leaders to execute the transforms function.
+Data transforms take data from an input topic and map it to multiple output topics. For each topic partition, a leader is responsible for handling the data. Redpanda runs a virtual machine (VM) on the same CPU core (shard) as these partition leaders to execute the transforms function.
```

Comment:
@kbatuigas  

"one ore more output topics".  It's not always multiple.

"Web Assembly Virtual Machine" to be specific